src/shared/components/home/site-form.tsx

```diff
@@ -1196,9 +1194,10 @@ export class SiteForm extends Component<SiteFormProps, SiteFormState> {
   }
 
   handleTaglineChange(i: SiteForm, index: number, val: string) {
-    let taglines = i.state.siteForm.taglines.unwrap();
-    taglines[index] = val;
-    i.state.siteForm.taglines = Some(taglines);
+    i.state.siteForm.taglines.match({
+      some: tls => { tls[index] = val; },
+      none: void 0
+    });
```

Comment:
Make sure to test this, because I'm not totally sure that it works.

Reply:
Yup, i've confirmed it working